test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -154,20 +183,17 @@ private static void ReplaceTestProjectOutputHostInTestProjectFixture(TestProject
         {
             var dotnet = testProjectFixture.BuiltDotnet;
 
-            var testProjectHost = testProjectFixture.TestProject.AppExe;
             var testProjectHostPolicy = testProjectFixture.TestProject.HostPolicyDll;
             var testProjectHostFxr = testProjectFixture.TestProject.HostFxrDll;
 
-            if (!File.Exists(testProjectHost) || !File.Exists(testProjectHostPolicy))
+            if (!File.Exists(testProjectHostPolicy))
             {
                 throw new Exception("host or hostpolicy does not exist in test project output. Is this a standalone app?");
             }
 
-            var dotnetHost = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"dotnet{testProjectFixture.ExeExtension}");
             var dotnetHostPolicy = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostpolicy{testProjectFixture.SharedLibraryExtension}");
-            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
+            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionHostFxrPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
```

Comment:
> does not place correct hostpolicy is still functional in core-setup repo

Are you suggesting that publishing a standalone app, with sources as local folders that contain the built packages, does not restore the built hostpolicy? If so, we should fix it as part of this PR without adding debt of getting another change in and making it clean in a single go.